dags.yaml

```diff
@@ -0,0 +1,19 @@
+bqetl_error_aggregates:
+  schedule_interval: 3h
```

Comment:
There are a few places where `timedeltas` are used. Currently those can be written as a string like `3h`, `1d5h10m`, `30m`. For now only timedeltas with day, hour and minute components are supported.